seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json

```diff
@@ -252,26 +252,21 @@
       "description": [
         "Continuing the discussion on calling functions, the next bug to watch out for is when a function's arguments are supplied in the incorrect order. If the arguments are different types, such as a function expecting an array and an integer, this will likely throw a runtime error. If the arguments are the same type (all integers, for example), then the logic of the code won't make sense. Make sure to supply all required arguments, in the proper order to avoid these issues.",
         "<hr>",
-        "The following function <code>positivePowers</code> raises a base to a positive exponent. Unfortunately, it's not called properly - fix the code so the value of <code>power</code> is the expected 8."
+        "The following function <code>raiseToPower</code> raises a base to a positive exponent. Unfortunately, it's not called properly - fix the code so the value of <code>power</code> is the expected 8."
```

Comment:
@Manish-Giri - awesome job on this! Minor text fixes I didn't catch in the original issue:

Delete the word "following" in the instructions (it sounds weird since the function is to the right in the code editor). Then rephrase "... raises a base to a positive exponent..." to "... raises a base to an exponent..."

Then this looks ready to merge!